openmc/mgxs/mgxs.py

```diff
@@ -2452,7 +2866,7 @@ def __init__(self, domain=None, domain_type=None,
         self._rxn_type = 'nu-scatter'
 
 
-class ScatterMatrixXS(MGXS):
+class ScatterMatrixXS(MatrixMGXS):
```

Comment:
But no I haven't re-run those notebooks. I suppose I should eventually get OpenMOC running.